docs/UserGuide.md

```diff
@@ -305,63 +344,97 @@ To edit the tags of a contact, you can refer to [adding a tag](#744-adding-a-tag
+![And Condition Search](images/AndConditionSearch.png)
 
-Format: Refer to the [`search`](#724-searching-for-a-contact-search) command format above.
+**2. `or` Condition Search:**
+
+If you wish to broaden your search result, you can use `or` condition search to search for contacts that have information matches **at least one** of the parameters given.
+
+Format: `search or [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [t/TAG]…`
+
+* The search using `n/NAME` is case-insensitive. (e.g. `hans` will match `Hans`).
+* At least 1 of the optional parameters must be provided.
+
+**Example Input in Command Box**
+```
+`search or n/Bernice t/cs2100 t/friends`
+```
+
+**Example Result**
 
-* This feature is only available when search command is entered (i.e. the command entered matches the [`search`](#724-searching-for-a-contact-search) format stated above).
-* Only the last parameter will be completed (e.g. `search and n/NAME p/PHONE`, only the last parameter `PHONE` will be completed).
-* The list of search queries will include the current search query.
-* No result will be displayed if there is no contact matches the current search query.
-* No result will be displayed if the last parameter is empty.
+![Or Condition Search](images/OrConditionSearch.png)
 
-Example:
-* `search or n/John p/` displays nothing as the last parameter `PHONE` is empty.
-* `search and n/John a/N`, displays a list of search queries containing `search and n/John a/N` and `search and n/John a/NUS` if SoConnect has contacts with name `John` and address `NUS`
-* `search and n/John a/N` displays nothing if SoConnect does not have any contact with name `John` or has contact with name `John` but does not start with `N`.
-* `search or n/John a/N` displays a list of search queries containing `search and n/John a/N`, `search and n/John a/NUS`, `search and n/John a/NYC` if SoConnect has contacts with address `NTU` and `NYC`, does not have to care about the name in the contact since it is `or` condition.
-* `search or n/John a/N` displays nothing if SoConnect does not have contacts with address starts with `N`.
+Notice there are more contacts listed as compared to `search and n/Bernice t/cs2100 t/friends`.
+
+<div markdown="block" class="alert alert-primary">
+**:memo: Note:**<br/>
+For both `and` and `or` condition search, you still get a list of relevant contacts related to your search query if no search result available. This is useful as you might still get the contacts that you are searching for when you entered some characters wrongly.<br/><br/>A contact is considered relevant if there are high matches of characters between the contact information and search parameters. For example, the information `David Li` and `Charlotte` are relevant to `search n/al` because these names contain characters `a` and `l` in it.
```

Comment:
```suggestion
For both `and` and `or` condition search, you still get a list of relevant contacts related to your search query if there are no search results available. This is useful as you might still get the contacts that you are searching for when you entered some characters wrongly.<br/><br/>A contact is considered relevant if there are high matches of characters between the contact information and search parameters. For example, the information `David Li` and `Charlotte` are relevant to `search n/al` because these names contain characters `a` and `l` in it.
```